docs/forms.md

```diff
@@ -0,0 +1,59 @@
+# Forms
+
+Our `RapidRailsFormBuilder` is a custom form builder class that provides additional functionality and customization for form fields.
```

Comment:
link tot he RapidRailsFormBuilder file from here

Reply:
say that it build on top of the default Rail's form builder and that they can customise the styling of the forms and things from there if they wish. but they can also decide not to use the form builder and go the default rails form route and style things themselves in the form